templates/mets_asset.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <METS:mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:METS="http://www.loc.gov/METS/v2" xmlns:PREMIS="http://www.loc.gov/premis/v3" OBJID="{{ object_identifier }}" xsi:schemaLocation="http://www.loc.gov/METS/v2 ../v2/mets.xsd http://www.loc.gov/premis/v3 http://www.loc.gov/standards/premis/v3/premis.xsd">
 
-  <METS:metsHdr RECORDSTATUS="{{ action }}" CREATEDATE="{{ create_date }}" ID="HDR1">
+  <METS:metsHdr RECORDSTATUS="{{ action }}" CREATEDATE="{{ create_date }}" ID="HDR1" TYPE="Asset">
     <METS:agent ROLE="CREATOR" TYPE="ORGANIZATION">
       <METS:name>University of Michigan - Library Information Technology - Digital Collection Services</METS:name>
     </METS:agent>
+    <METS:altRecordID TYPE="DLXS">{{ alternate_identifier }}</METS:altRecordID>
```

Comment:
Is there a reason we want to use "DLXS" here instead of "DOR"?

Reply:
The alternate identifier reflects the "local convention", however we signal that going forward, e.g. this monograph has a UUID but is known locally as `act3868.0027.001`. The "local convention" probably won't be "DLXS" in the end, but it's an obvious type as we develop.